plugin/hover.py

```diff
@@ -148,16 +147,22 @@ def diagnostics_content(self) -> str:
                     format_diagnostic_for_html(self.view, diagnostic, self._base_dir))
             for items in by_severity.values():
                 formatted.extend(items)
-            config_name = sb.session.config.name
-            if config_name in self._actions_by_config:
-                action_count = len(self._actions_by_config[config_name])
-                if action_count > 0:
-                    href = "{}:{}".format('code-actions', config_name)
-                    text = "choose code action ({} available)".format(action_count)
-                    formatted.append('<div class="actions">[{}] {}</div>'.format(config_name, make_link(href, text)))
             formatted.append("</div>")
         return "".join(formatted)
 
+    def code_actions_content(self) -> str:
+        formatted = []
+        for config_name, actions in self._actions_by_config.items():
+            action_count = len(actions)
+            if action_count > 0:
+                href = "{}:{}".format('code-actions', config_name)
+                if action_count > 1:
+                    text = "choose code action ({} available)".format(action_count)
+                else:
+                    text = actions[0].get('title', 'code action')
+                formatted.append('<div class="actions">[{}] {}</div>'.format(config_name, make_link(href, text)))
+        return "".join(formatted)
```

Comment:
Can this go back to where it was in `diagnostics_content`? (but with the `action > 1` check)

Reply:
The `code action` does not just occur during the diagnosis.Please see my screenshot